src/components/DisplayResources.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+import { batchCreateResources } from "@api/resources/route.client";
+import { batchCreateRequestSchema } from "@api/resources/route.schema";
```

Comment:
I think we will handle modifying resources here, but should remove batchCreateRequestSchema